flowkit/flowkit.go

```diff
@@ -864,16 +864,29 @@ func (f *Flowkit) GetTransactionsByBlockID(
 	_ context.Context,
 	blockID flow.Identifier,
 ) ([]*flow.Transaction, []*flow.TransactionResult, error) {
+
 	tx, err := f.gateway.GetTransactionsByBlockID(blockID)
 	if err != nil {
 		return nil, nil, err
 	}
-
 	txRes, err := f.gateway.GetTransactionResultsByBlockID(blockID)
-	if err != nil {
-		return nil, nil, err
+	if err == nil {
+		return tx, txRes, nil
+	}
+	errorMessage := err.Error()
+
+	if strings.Contains(errorMessage, "received message larger than max") || strings.Contains(errorMessage, "trying to send message larger than max") {
```

Comment:
do we have access to these errors? so we could import them by type instead of hardcoding error message?

Reply:
they look like standard GRPC error messages to me, so i doubt it.